js/data/question.jsonc

```diff
@@ -0,0 +1,599 @@
+	"import_to_main_fleet": "Import to Main Fleet", // ?
+	"import_to_friend_fleets": "Import to Friend Fleets", // ?
+	"load_from_dewy": "Dewyから読み込み",
+	"load_from_kcnav": "KCNavから読み込み",
+	"no_data_found": "データが見つかりませんでした",
+    "no_ship_matches_found": "該当する船舶がありません",
+	"request_error_not_found": "Request error/Not found", // ok
+	"request_timed_out": "Request timed out", // ok
+	"ship": "艦",
+    "count": "Count", // ?
+    "max_low": "最大",
+    "min_high": "最小",
+    "import_value": "インポート",
+    "low_all": "最低値を採用",
+    "high_all": "最高値を採用",
+    "average_all": "平均値を採用",
+	"confirm": "決定",
+	
+	"warn_replay_import": "Warning: Replay import will overwrite existing data.", // ?
+    "download_backup_of_current": "現在の艦隊とNodeデータのバックアップをダウンロードしますか？",
+    "download_a_backup": "現在の艦隊とNodeデータのバックアップをダウンロードします。",
+	"download": "Download", // ok
+	"load_backup": "Load Backup", // ok
+	"error": "エラー",
+    "import_all": "読み込んでページを再読み込み",
+	"import_player_only": "自艦隊と基地航空隊のみインポート",
+    "reset_all_data": "データを全て削除してページを再読み込み",
+	"reset_confirm": "確認",
+	"continue": "再試行",
+	
+	"require_los_after_fcf": "退避後に必要な索敵値",
+    // require_on_node could be "Node: [input] において", which is natural.
+	"require_on_node": "on Node",
+	"node_all": "All", // ok
+	"require_radar_after_fcf": "退避後に必要な水上電探の数",
+	"require_ship_counts": "退避後に必要な艦数及び艦種",
+	"ship_counts_desc": "例: \"DD\", \"DE/DD/CL\", \"XX\"",
+	"add": "決定",
+	"never_use_on": "離脱を許可しない艦",
+    "note_require_on_node": "「Node: [Node番号] において」は指定したNodeとそれより前の全てのNodeに対して適用します。例.Node番号に2を指定: Node1とNode2に適用し、Node3以降には適用しない",
+	
+	"auto_set_historical_bonus": "史実ボーナスを各Nodeへまとめて指定",
+    "current_data_set_updated": "現在適用しているデータセットが{date}に更新されました。今すぐ全てのボーナスを再適用しますか?",
+	"manually_compiled_bonuses": "手動で編集したボーナス(可能であれば推奨)",
+	"simulator_preset": "Simulator Preset",
+    "bonus_ranges_auto_calc": "TsunDBから自動計算されるボーナス範囲",
+	"set": "Set", // ok
+	"none": "None", // ok
+	"note": "メモ",
+	"damage": "ダメージ",
+	"use_preset_values": "プリセットの値を使用",
+	"use_average": "平均値を使用",
+	"use_min": "最小値を使用",
+	"use_max": "最大値を使用",
+	"accuracy_evasion": "命中/回避",
+	"none_ignore": "なし (無効)",
+	"none_overwrite": "なし (上書き)",
+	"use_estimated_values": "推定値を使用",
+	"use_equal_damage_mod": "ダメージ補正と同じ値を使用",
+    "use_half_damage_mod": "ダメージ補正の半分の値を使用",
+	"include_speculated_ships": "予想される艦を含める",
+    "include_speculated_ships_desc": "まだデータが確認されていないが、過去の編成からボーナスがあると推測される艦",
+	"start": "決定",
+	"dismiss": "破棄",
+	"apply_to_current": "現在の編成に適用",
+    "apply_to_current_desc": "\u2610 これから新しく追加されるすべての艦は、デフォルトでこれらのボーナスを付与されます。\n\u2611 また、これらのボーナスを現在設定されているすべての船に適用します。(ONにしない場合、全て削除)",
+    "apply_dmg_only": "ダメージのみ適用",
+	"loading": "Loading", // ok
+	"load_error_autobonus_turn_off": "ロードエラー: 自動ボーナス設定がOFFになります",
+	
+	"mechanic": {
+		"artillerySpotting": "弾着観測射撃",
+		"AACI": "対空CI",
+		"fitGun": "フィット砲",
+		"OASW": "先制対潜",
+        "CVCI": "昼戦空母CI",
+		"destroyerNBCI": "駆逐艦の夜戦CI",
+		"LBASBuff": "LBAS/Support Revamp (Fall 17)", // ?

```

Comment:
Starting from Fall 2017 Event update, support and LBAS mechanics were updated to make them stronger: Support usable on night-only nodes, ASW support implemented, LBAS received bonus from proficiency, some calculation differences